### DISCUSSION TITLE
Icon Request - ATV

### DISCUSSION THREAD

**Discussion by yukisho** [FortAwesome/Font-Awesome#19872] [Feature Requests]
We have a Monster Truck and now a Bicycle, any chance on getting an ATV?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Please fill out our [new icon request template](https://github.com/FortAwesome/Font-Awesome/issues/new?assignees=&labels=new+icon&template=000_icon_request.yml&title=Icon+request%3A+)
